packages/loopring_v3/contracts/amm/AmplifiedAmmController.sol

```diff
@@ -126,14 +125,15 @@ contract AmplifiedAmmController is IAmmController, Claimable
 
     function consumeCurveChangeAuthorized(address pool)
         internal
-        returns (bool)
+        returns (bool authorized)
     {
         uint timestamp = curveChangeAuthorization[pool];
-        bool authorized = (timestamp <= block.timestamp) && (block.timestamp <= timestamp + MIN_CURVE_CHANGE_DELAY);
+        authorized = (timestamp <= block.timestamp) && 
+            (block.timestamp <= timestamp + MIN_CURVE_CHANGE_DELAY);
```

Comment:
Oops seems like I made a small mistake here (which currently doesn't matter because they have the same value :): `MIN_CURVE_CHANGE_DELAY` -> `CURVE_CHANGE_AUTH_WINDOW`.

Reply:
I‘ll change it.